### DISCUSSION TITLE
Is there a standalone dashboard package we can deploy to any k3s cluster as a replacement of Kubernetes dashboard

### DISCUSSION THREAD

**Discussion by aDisplayName** [rancher-sandbox/rancher-desktop#6490] [General]
We know the dashboard used by Rancher server cluster view and Rancher Desktop Dashboard is the same. 
Is there a package available to be deployed as a standalone helm app so that we can provide the same look'n'feel to manage the k3s cluster?

**Comment by jandubois** [comment]
Not sure if this is what you are asking, but you can deploy Rancher Manager locally on Rancher Desktop with something like this:

```shell
helm repo add jetstack https://charts.jetstack.io
helm repo add rancher-latest https://releases.rancher.com/server-charts/latest
helm repo update

helm upgrade \
     --install cert-manager jetstack/cert-manager \
     --namespace cert-manager \
     --set installCRDs=true \
     --set "extraArgs[0]=--enable-certificate-owner-ref=true" \
     --create-namespace

helm upgrade \
     --install rancher rancher-latest/rancher \
     --version "2.8.1" \
     --namespace cattle-system \
     --set hostname="localhost" \
     --wait \
     --timeout=10m \
     --create-namespace
```

This should work on macOS/Linux and let you login with https://localhost/dashboard/auth/login. On Windows you would need to use the actual load balancer IP instead.
